deploy/sharedAbiConfig.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
can this be json instead of js?  and make it an array of { localContractName: string, sharedContractName: string }?

Reply:
Sure no problem. Just out of curiosity, I didn't think you would be using this file on your end, or will you?

Reply:
Pardon, I just noticed that I actually cannot change the file to JSON without moving it to a different folder, because hardhat won't allow JSON files in the `deploy` folder. As far as I understand when we run the deploy command, hardhat  goes through all exported modules of the `deploy` folder and calls them. I thought it would just skip JSON files, but apparently it doesn't and throws an error instead. I could:
1. make the config part of the [export script](https://github.com/PrimeDAO/contracts-v2/blob/69-chore-hardhat-export-for-artifacts/deploy/99_export_abis.js)
2. put the JSON config on the root directory (or elsewhere)
3. leave it as it is

My preference would be 3., because 1. feels like hiding configuration in actual logic and since the config is only needed for the deploy functions in the `deploy` directory, I feel the `deploy` directory is the best place to host it.

Reply:
hmmm, maybe you are right.  so forget my request for now.